articles/tech-resources/enabling-multiple-custom-tracking-domains.md

```diff
@@ -47,6 +47,8 @@ To configure one or more custom tracking domains through the SparkPost user inte
     ![](media/enabling-multiple-custom-tracking-domains/xU06kJ2lac6YGEIi8KlReq8OaF1kfX.png)
 1. Complete the Sending Domain Name for each sending domain and select the Linked Tracking Domain you want to associate with it.
 1. Click the Add Domain or _Update Tracking Domain_ button to complete the association.
+Please note in order to associate a tracking domain with a sending doamin you need to make you have verified the trakcing domain first. 
+If you want to remove a trakcing domain you can simply go to the tracking domain section in your dashboard and click on the trash icon on the right hand side. 
```

Comment:
Can we combined this note with the other notes at the bottom?

4. Click the Add Domain or Update Tracking Domain button to complete the association. 

**Note:** For information on how to configure your Sending Domains, please see this article.  

**Note:** Any newly configured tracking domains have to route back, either through delegation or use of CNAME records, in order for us to be able to report on opens and clicks. Misconfiguration of your tracking domain(s) could mean link redirection could fail, so please ensure you test prior to placing any new domains into production!

**Note:** Deleting a tracking domain will automatically disassociate it from all sending domains.